bindings/gumjs/runtime/hexdump.js

```diff
@@ -11,6 +11,7 @@ function hexdump(target, options) {
   const useAnsi = options.hasOwnProperty('ansi') ? options.ansi : false;
 
   let buffer;
+  let startAddress = 0;
```

Comment:
```suggestion
  let startAddress = NULL;
```

Reply:
What happens if the address is from an array, not a pointer? NULL + offset in Javascript?

Reply:
It's just a pointer value used for display purposes, i.e. it's never dereferenced, so it's safe also in the `NULL` case.

Reply:
You are far the better programmer, so I believe you ;).